server/routes/pages.js

```diff
@@ -32,13 +32,76 @@ module.exports = function(server)
                 footer: 'partials/footer',
             },
             stars: helper.generateStars(40),
-            clouds: helper.generateClouds(),
+            clouds: helper.generateClouds(1),
```

Comment:
Why does an argument need to be passed to this function?

Reply:
oops i missed that, i was trying to figure out how the clouds worked and I needed the deps.js file to be built